src/hdskins/java/com/minelittlepony/gui/Button.java

```diff
@@ -0,0 +1,47 @@
+package com.minelittlepony.gui;
+
+import java.util.List;
+
+import com.google.common.base.Splitter;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+
+public class Button extends GuiButton implements IActionable, IGuiTooltipped {
+
+    protected IGuiAction<Button> action;
+
+    private List<String> tooltip = null;
+
+    @SuppressWarnings("unchecked")
+    public Button(int x, int y, int width, int height, String label, IGuiAction<? extends Button> callback) {
+        super(5000, x, y, width, height, GameGui.format(label));
```

Comment:
You can't just give everything the same ID. This is one reason why when I made my gui framework, everything extends `Gui` instead of `GuiButton`. It limits the risk of mod conflicts.

Reply:
These buttons aren't even using the button for anything.

Reply:
The buttons aren't, but it still exists as a field that other mods might check.

Reply:
Well the entire purpose of these classes even existing is to remove the need to specify a unique id for each one. You're essentially calling out a glass of milk for being white.

Reply:
No, I'm calling out a glass of chocolate milk for being white.